src/datastore/src/userstore.js

```diff
@@ -51,7 +51,7 @@ class UserStore extends NetworkStore {
     }
 
     if (!data._id) {
-      return Promise.ject(new KinveyError('User must have an _id.'));
+      return Promise.reject(new KinveyError('User must have an _id.'));
```

Comment:
A [unit test](https://github.com/Kinvey/js-sdk/pull/101/files#diff-ea2fab4905bfc00b94d59bb6949998e2R306) has already been written to cover this case.